### DISCUSSION TITLE
edl import possible?

### DISCUSSION THREAD

**Discussion by MCal27** [mifi/lossless-cut#667] [Ideas]
I'm using a media server product which automatically creates an edl file alongside the video output file. I can't see an option to import edl files to assist with accurate and quick cuts. Am I missing something or can this function be considered for a future version?

Thanks.

**Comment by markusd1984** [comment]
That would be a new feature request for an **Import Project Profile**, might be worthwhile sharing the format used in the [edl](https://en.wikipedia.org/wiki/Edit_decision_list) (which I know is popular for cutting out commercials).

**Comment by mifi** [comment]
do you mean like this? https://github.com/mifi/lossless-cut/issues/609
or what kind of EDL format are you thinking about

**Reply by MCal27** [reply]
yes exactly that...  sorry missed that it had already been asked.

**Reply by mifi** [reply]
This is now implemented and will be included in the next version

**Reply by MCal27** [reply]
Thankyou!
